packages/run-protocol/src/psm/psm.js

```diff
@@ -42,9 +42,9 @@ function stageTransfer(from, to, txFrom, txTo = txFrom) {
 
 /**
  * @param {ZCF<GovernanceTerms<{
- *    GiveStableFee: ParamRecord<'ratio'>,
- *    WantStableFee: ParamRecord<'ratio'>,
- *    MintLimit: ParamRecord<'amount'>,
```

Comment:
the type spec doesn't need the full record with the `value` field that was implied by the `type` field